app/Filament/Resources/Financial/AccountsReceivables/Actions/MarkAccountsInstallmentAsReceivedAction.php

```
@@ -0,0 +1,98 @@
+            ->title('Parcela marcada como recebida!')
+            ->success()
+            ->send();
+    }
+
+    /**
+     * Create the next installment for recurring accounts.
+     */
+    private static function createRecurringInstallment(AccountsInstallments $installment, $account): void
+    {
+        $newDueDate = $installment->due_date->copy()->addMonth();
+        $newInstallmentAmount = $installment->amount;
```

PHPStan (2.1.33)

Access to an undefined property App\Models\Accounts\AccountsInstallments::$amount.
Learn more: https://phpstan.org/blog/solving-phpstan-access-to-undefined-property

(property.notFound)